src/contacts/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Google LLC
+# Copyright 2021 Google LLC
```

Comment:
General question: do we typically modify the copyright year per-file on-change, or do we try to keep all of them up-to-date as much as possible?

Reply:
I try to manually update to the current year anytime I change the file. The `dev-kubernetes-manifests` have to get manually updated for the new year, before generating the next release